interface/super/edit_globals.php

```diff
@@ -645,6 +645,22 @@ function checkBackgroundServices()
                                                     echo "</option>\n";
                                                 }
                                                 echo "  </select>\n";
+                                            } elseif ($fldtype == GlobalSetting::DATA_TYPE_MULTI_DASHBOARD_CARDS) {
+                                                $res = sqlStatement("SELECT * FROM dashboard_card_visibility");
+                                                echo "  <select multiple class='form-control' name='form_{$i}[]' id='form_{$i}[]' size='3'>\n";
+                                                while ($row = sqlFetchArray($res)) {
+                                                    echo "   <option value='" . attr($row['card_abrev']) . "'";
```

Comment:
We need a default none or empty so user can select to hide 0 cards. Currently once you select cards you will always have to select at least one card.
`<option value="">attr("Hide None")</option>`
Or something like after select echo.

Reply:
actually I suppose a ctrl click will work so maybe okay the way you have it.
Thanks for the effort Sherwin.